### PR TITLE
HTTP to HTTPS

### PR DESCRIPTION
**Pull request by Spekular**
Change some links to HTTPS.

### CODE REVIEW

File: public/feed/soundcloud.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
 		$item->user->username, 			// $author	i.e. "John Smith"
 		$item->user->permalink_url, 	// $author_href	i.e. "http://facebook.com/user1234"
 		$item->created_at,				// $date	i.e. "2014-01-01 00:00:00"
-		($item->artwork_url ? $item->artwork_url : $item->user->avatar_url), // $thumbnail	i.e. "http://facebook.com/etc.png"
+		($item->artwork_url ? $item->artwork_url : $item->user->avatar_url), // $thumbnail	i.e. "https://facebook.com/etc.png"
```
Comment (tresf) [inline public/feed/soundcloud.php]: 😠

File: public/feed/youtube.php [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 
 		$id = parse_youtube_id($item->thumbnails->default->url);
-		$url = $id ? 'http://www.youtube.com/watch?v=' . $id : '';
+		$url = $id ? 'https://www.youtube.com/watch?v=' . $id : '';
```
Comment (tresf) [inline public/feed/youtube.php]: The youtube iframe does this, no need to change.

**Comment by Spekular** [comment]
Dunzo

File: templates/home.twig [hunk 3]
```diff
@@ -12,7 +12,7 @@
 		<div class="container">
 
 			<div class="visible-lg visible-md pull-right jumbotron-video">
-			{{ youtube_iframe('http://youtu.be/W6tEolVz3_4', 356, 200, '?showinfo=0') }}
+			{{ youtube_iframe('https://youtu.be/W6tEolVz3_4', 356, 200, '?showinfo=0') }}
```
Comment (tresf) [inline templates/home.twig]: The iframe code handles HTTP vs HTTPS automatically.

Comment (Spekular) [inline templates/home.twig]: done

**Comment by tresf** [comment]
Thanks!  If you're really kind, you'll do the [`testing`](https://github.com/LMMS/lmms.io/tree/testing) branch too as @Umcaruje has sure to touched all of these and it would otherwise cause a merge conflict. 👍

**Comment by Spekular** [comment]
Maaaaybe tomorrow. :P

**Pull request merged**